### DISCUSSION TITLE
Converting TexturesVertex to TexturesUV / save as .mtl file?

### DISCUSSION THREAD

**Discussion by mark-at-hypothetic** [facebookresearch/pytorch3d#1449] [General]
The goal is saving my mesh's vertex colors as a texture .mtl file. 

I guess, a logical step would be first to convert the colors from TexturesVertex to TexturesUV? Is that possible? 

I've seen it being done the other way around in [this](https://github.com/facebookresearch/pytorch3d/issues/854) discussion, but haven't found a way to do the reverse yet.

If impossible, anyone know of a tool that does it which I can embed in my python pipeline? I think baking vertex colors to the mesh is probably possible in the Blender Python API but haven't gotten that to work yet either.

**Comment by bottler** [comment]
There's no particular facility in PyTorch3D to help you do this. You need to come up with locations of each face in the UV-plane somehow.

**Reply by mark-at-hypothetic** [reply]
Alright. If anyone has a good implementation for that I'd be interested to hear
